packages/discord.js-utilities/src/lib/PaginatedMessage.ts

```diff
@@ -225,9 +216,43 @@ export class PaginatedMessage {
 	public clone() {
 		const clone = new PaginatedMessage({ pages: this.pages, actions: [] }).setIndex(this.index).setIdle(this.idle);
 		clone.actions = this.actions;
+		clone.response = this.response;
+		clone.collector = this.collector;
 		return clone;
 	}
 
+	private async handleCollect(author: User, channel: TextChannel | NewsChannel, reaction: MessageReaction, user: User) {
```

Comment:
Maybe protected so this shows up when extending the class for easier overwriting? Same for handleEnd.